### PR TITLE
#3438 add sign isha step

### PR DESCRIPTION
**Pull request by desfero**
**Technical notes**
`selectEtoStep` was moved to use a single object as param

### CODE REVIEW

File: packages/web/app/test-e2e/utils/index.ts [hunk 1]
```diff
@@ -231,11 +231,6 @@ export const loginWithLightWallet = (email: string, password: string) => {
   lightWalletTypeLoginInfo(email, password);
 };
 
-export const acceptWallet = () => {
```
Comment (desfero) [inline packages/web/app/test-e2e/utils/index.ts]: Duplicate of `confirmAccessModal`

File: packages/web/intl/locales/en-en.json [hunk 2]
```diff
@@ -195,6 +196,7 @@
   "eto-dashboard.review": "Investment Offer Is Being Published",
   "eto-dashboard.review.description": "Your investment offer is being published as we speak. Someone from our team will coordinate with you the exact publishing time. Once your investment offer is live you will be able to refer investors to it and enable whitelisting.",
   "eto-dashboard.setup-start-date": "Agree To Terms Of Your Eto And Set The Date",
+  "eto-dashboard.sign-your_isha": "Sign Your  Investment And Shareholder Agreement",
```
Comment (AlexeyTuboltsev) [inline packages/web/intl/locales/en-en.json]: double space between `Your` and `Investment`

Comment (desfero) [reply packages/web/intl/locales/en-en.json]: good catch, fixed

File: packages/web/app/components/documents/issuerDocuments/Documents.tsx [hunk 3]
```diff
@@ -100,9 +100,9 @@ const Documents = compose<TComponentProps, {}>(
     renderComponent(LoadingIndicator),
   ),
   appConnect<TStateProps, TDispatchProps, { etoId: string }>({
-    stateToProps: (state, ownProps) => ({
+    stateToProps: state => ({
       etoState: selectIssuerEtoState(state)!,
-      onChainState: selectEtoOnChainStateById(state, ownProps.etoId)!,
+      onChainState: selectIssuerEtoOnChainState(state)!,
```
[the remark this answers is not in the document]
Comment (desfero) [reply packages/web/app/components/documents/issuerDocuments/Documents.tsx]: wrapped into `nonNullable`

**Pull request merged**